cmd/interfaces.go

```diff
@@ -2,14 +2,15 @@ package main
 
 import (
 	"fmt"
+
```

Comment:
go doesn't treat base libs different from external, so adding space here is kinda counter productive

Reply:
strange; the empty line is inserted automatically by a vs code plugin. I think it don't harm ;) but maybe there is some place where it can be configured